roles/openshift_serviceaccounts/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Create service account configs
+  template:
+    src: serviceaccount.j2
+    dest: "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
+
+- name: Create {{ item }} service account
+  command: >
+    {{ openshift.common.client_binary }} create -f "/tmp/{{ item }}-serviceaccount.yaml"
+  with_items: accounts
```

Comment:
This fails if run more than once, add failed_when and changed_when so we can run it multiple times.
```
   register: _sa_result
   failed_when: "'serviceaccounts \"{{ item }}\" already exists' not in _sa_result.stderr and _sa_result.rc != 0"
   changed_when: "'serviceaccounts \"{{ item }}\" already exists' not in _sa_result.stderr and _sa_result.rc == 0"
```